.changeset/sharp-worms-chew.md

```diff
@@ -0,0 +1,8 @@
+---
+'@sap-ux/abap-deploy-config-sub-generator': minor
+'@sap-ux/deploy-config-sub-generator': minor
```

Comment:
`@sap-ux/deploy-config-sub-generator` does not need a changeset, it only has a test update